### PR TITLE
`umd` and `es` targets instead of import source

### LINKED ISSUE #233: `createLogger` doesn't seem to be exported correctly in latest build
Just updated the 3.0.6 and getting errors in rollup saying:

`Error: 'createLogger' is not exported by node_modules/redux-logger/dist/redux-logger.js`

We can solve this using `namedExports` in the rollup-plugin-commonjs module but I think a better solution would be to readd js:next to package.json. If users are able to consume the ES2015 version it will allow apps using rollup to have tree shaking magic

### PR DESCRIPTION
**Pull request by thiamsantos**
That should close #233.

But needs testing to prevent #229 and #231 to happen again.

### CODE REVIEW

**Comment by evgenyrodionov** [comment]
Can you test?

**Comment by thiamsantos** [comment]
Sure. Right now I can't, but I will do it when I get home.

**Comment by thiamsantos** [comment]
Sorry for the delay, I will try to test it in this weekend.

**Comment by thiamsantos** [comment]
I tested with rollup and webpack,  and everything seems to work.

Dependencies used:
```js
{
  "babel-core": "^6.24.1",
  "babel-loader": "^7.0.0",
  "babel-plugin-external-helpers": "^6.22.0",
  "babel-preset-env": "^1.4.0",
  "rollup": "^0.41.6",
  "rollup-plugin-babel": "^2.7.1",
  "rollup-plugin-commonjs": "^8.0.2",
  "rollup-plugin-node-resolve": "^3.0.0",
  "webpack": "^2.4.1"
}
```

`webpack.config.js`
```js
var path = require('path')

module.exports = {
  entry: './app/main.js',
  output: {
    filename: 'bundle.js',
    path: path.resolve(__dirname, 'public')
  },
  module: {
    rules: [
      {
        test: /\.js$/,
        exclude: /node_modules/,
        use: {
          loader: 'babel-loader',
          options: {
            presets: ['env']
          }
        }
      }
    ]
  }
}
```

`rollup.config.js`
```js
import commonjs from 'rollup-plugin-commonjs'
import nodeResolve from 'rollup-plugin-node-resolve'
import babel from 'rollup-plugin-babel'

export default {
  entry: 'app/main.js',
  format: 'iife',
  plugins: [
    babel({
      exclude: 'node_modules/**',
      presets: [
        [
          'env',
          {
            modules: false
          }
        ]
      ],
      plugins: [
        'external-helpers'
      ]
    }),
    nodeResolve({
      jsnext: true,
      module: true,
      main: true,
      browser: true
    }),
    commonjs()
  ],
  dest: 'public/bundle.js'
}
```

**Comment by maxrbaldwin** [comment]
Still getting the `Module not found: Error: Can't resolve 'redux-logger'` error. This seems to be the fix. Will this get merged any time soon?

File: package.json [hunk 1]
```diff
@@ -38,8 +41,7 @@
     ]
   },
   "files": [
-    "dist",
-    "src"
```
Comment (evgenyrodionov) [inline package.json]: Can you bring back `src` folder? I think some users can rely on this and I don't see reasons break it.

Comment (thiamsantos) [inline package.json]: Sure

Comment (thiamsantos) [inline package.json]: Done

**Comment by thiamsantos** [comment]
Any thoughts @evgenyrodionov?

**Comment by deadlybutter** [comment]
Any updates on this being merged?

**Comment by thiamsantos** [comment]
Any updates on this @evgenyrodionov ?

**Comment by grushetsky** [comment]
@thiamsantos, could you, please, resolve conflicts by rebasing to the latest branch?

**Comment by thiamsantos** [comment]
@grushetsky conflicts resolved.

**Comment by grushetsky** [comment]
@evgenyrodionov, let's merge this one, do you mind?

**Comment by evgenyrodionov** [comment]
+

**Comment by gregorym** [comment]
Has this been released in 3.0.6? I'm still seeing this issue with version 3.0.6

**Comment by adipascu** [comment]
The fix is not currently released.

**Comment by gregorym** [comment]
@evgenyrodionov Is something preventing a new release (`3.0.7`)? This has been merged months ago.

**Comment by d4rky-pl** [comment]
Bump

**Pull request merged**